torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -308,7 +314,8 @@ class ChunkDataset final
         example_sampler_(std::move(example_sampler)),
         options_(std::move(options)),
         quit_worker_(false),
-        running_preloaders_(0) {}
+        running_preloaders_(0),
+        load_checkpoint_(!options_.resume_from_file_.empty()) {}
```

Comment:
Should we do some input validation on resume_from_file_ string, to make sure it is a valid filename or that we have read/write access to it?

Reply:
actually pytorch's save/load method takes care of it. If the fire is not accessible, those method will error out. So I think we are good here :)